.github/workflows/cron.yml

```diff
@@ -1,11 +1,12 @@
 ---
 name: Trivy Java DB
 on:
+  workflow_dispatch:
```

Comment:
@tejadf  we removed this earlier to avoid workflow triggers manually right?

Reply:
You can address this under https://df-eng.atlassian.net/browse/DEEP-10028